creme/billing/tests/test_export.py

```diff
@@ -62,6 +65,7 @@ def test_invoice(self):
 
     @skipIfCustomQuote
     @skipIfCustomServiceLine
+    @skipIf(which("pdflatex") is None, "pdflatex is not installed.")
```

Comment:
This is temporary, I think we planned to replace pdflatex in a near release ?

Reply:
not replace it, but rather add new ones.

Reply:
Yes the next big feature should be a backend system for billing export.

1. 'pdflatex' will be one the backends, & will probably be excluded from the default ones (because installing Latex is a pain in the ***.
2. I'll probably rework this test with a mocking backend anyway.